drivers/include/ata8520e.h

```diff
@@ -82,14 +82,14 @@ typedef enum {
  * See ATA8520E datasheet, section 2.1.2.10, page 12
  */
 typedef enum {
-    ATA8520E_ATMEL_OK,                       /**< No error */
-    ATA8520E_ATMEL_COMMAND_ERROR,            /**< Command error / not supported */
-    ATA8520E_ATMEL_GENERIC_ERROR,            /**< Generic error */
-    ATA8520E_ATMEL_FREQUENCY_ERROR,          /**< Frequency error */
-    ATA8520E_ATMEL_USAGE_ERROR,              /**< Usage error */
-    ATA8520E_ATMEL_OPENING_ERROR,            /**< Opening error */
-    ATA8520E_ATMEL_CLOSING_ERROR,            /**< Closing error */
-    ATA8520E_ATMEL_SEND_ERROR                /**< Send error */
+    ATA8520E_ATMEL_OK,                      /**< No error */
```

Comment:
Please revert the doxygen 1 character shift of this enum, it seems unrelated.

Reply:
no, it is not, see the first commit of this PR: https://github.com/RIOT-OS/RIOT/pull/8449/commits/acac4290f5236c8bb16e267eb4813414e5b4de3e

Alignment in the master was off for one space for this block...

Reply:
> Alignment in the master was off for one space for this block...

I don't understand. Are you still talking of the "multiple of 4" indentations requirements? I don't consider this as a rule of thumb for alignment, thus the doxygen comment alignment is not 'off' in master. Maybe you should update your editor configuration ?

Reply:
I'm refering this comment: "some doxygen blocks were not aligned to 4 spaces (this upset my text editor :-) )"

Reply:
Yes, I am still talking about the 4-space alignment. In master, the this doxygen block is indented by 45 characters and `45 % 4 != 0`. So I put moved it to 44 characters -> now 4-space aligned.

> I don't consider this as a rule of thumb for alignment

But the coding convention does per default, as long as there is no good reason not to do so. I always interpreted this also applicable to doxygen blocks -> so yes, I say the 4-space alignment does apply here.

Furthermore, even if ignoring the 4-space alignment rule, it does not make sense to indent the block in questions here in a different style than the two blocks below...

Reply:
@haukepetersen what about stopping discussing what I'm requested and revert this change ? Those doxygen comments are aligned on their own and the code is nice enough (and not only because I wrote it ;) ). 

The only meaningful change of this PR is related to the documentation of payload sizes that are incorrect. Only this should be changed I think.

Reply:
I think it is completely valid to piggy-back these kind of formatting fixes (and in this case it is a fix) with a related PR if the change in question is kept in its own commit. This is the case here, so let's not block this PR on a matter of opinion.